toml.md

```diff
@@ -161,9 +161,17 @@ In JSON land, that would give you the following structure:
 }
 ```
 
-Whitespace around dot-separated parts is ignored, however, best practice is to
+Whitespace around dot-separated parts is ignored. However, best practice is to
 not use any extraneous whitespace.
 
+```toml
+fruit.name = "banana"     # this is best practice
+fruit. color = "yellow"    # same as fruit.color
+fruit . flavor = "banana"   # same as fruit.flavor
+```
+
+Indentation is treated as whitespace. Tabs and spaces before keys are ignored.
```

Comment:
```suggestion
Indentation is treated as whitespace. Whitespace before keys is ignored.
```

Reply:
Or even:
```suggestion
Indentation is treated as whitespace and ignored.
```

Reply:
I picked your second suggestion. It keeps the note about indentation and only repeats that.

Reply:
The same redundancy appeared under the dotted keys section as well, so I applied the same change there.